zerver/models.py

```diff
@@ -2531,8 +2543,11 @@ def get_huddle_backend(huddle_hash: str, id_list: List[int]) -> Huddle:
             huddle.recipient = recipient
             huddle.save(update_fields=["recipient"])
             subs_to_create = [Subscription(recipient=recipient,
```

Comment:
Can you clean up the code formatting here to comply with `black` style formatting (no hanging indent)?